library/src/morpheus-master/sass/base/_extendables.scss

```diff
@@ -279,6 +270,11 @@
 		}
 	}
 
+	&:focus {
+		outline:none;
+		box-shadow: $sakai-box-shadow-emulate-focus var(--focus-outline-color);
+	}
```

Comment:
This allows rounded elements to have a better looking focus which emulates our current `outline` focus